frontend/test/metabase/scenarios/dashboard-filters/dashboard-filters-source.cy.spec.js

```diff
@@ -20,7 +20,7 @@ const dashboardQuestionDetails = {
 };
 
 const structuredQuestionDetails = {
-  name: "GUI categories",
+  name: "Categories",
```

Comment:
Not quite sure what's the reason for changing both question names to the same name. I think the previous name is more explicit and easier to understand when reading each test individually.